setup.py

```diff
@@ -67,11 +68,15 @@
 if USE_CYTHON:
     extensions = cythonize(extensions)
 
+class bdist_rpm(_bdist_rpm):
+    def _make_spec_file(self):
+        orig = super()._make_spec_file()
+        orig.insert(4, '%define debug_package %{nil}')
```

Comment:
i just need it in the beginning, since apparently order of definition matters. First 4 were defined as
```
            '%define name ' + self.distribution.get_name(),
            '%define version ' + self.distribution.get_version().replace('-','_'),
            '%define unmangled_version ' + self.distribution.get_version(),
            '%define release ' + self.release.replace('-','_'),
```
But i guess putting it first won't really make a difference as no one will visually inspect the file anyway

Reply:
Usually `debug_package` is defined at the top, but that is simply convention.